test/integration/includes.cc

```diff
@@ -737,29 +747,23 @@ TEST(IncludesTest, InvalidMergeInclude)
     </sdf>)";
     sdf::Root root;
     sdf::Errors errors = root.LoadSdfString(sdfString, config);
-    ASSERT_FALSE(errors.empty());
-    EXPECT_EQ(sdf::ErrorCode::MERGE_INCLUDE_UNSUPPORTED, errors[0].Code());
-    EXPECT_EQ("Merge-include does not support parent element of type frame",
-              errors[0].Message());
-    EXPECT_EQ(5, errors[0].LineNumber());
+    ASSERT_EQ(3, errors.size());
+    EXPECT_EQ(sdf::ErrorCode::ELEMENT_INCORRECT_TYPE, errors[0].Code());
+    EXPECT_NE(std::string::npos, errors[0].Message().find(
+        "XML Element[alwaysOn], child of element[sensor], not defined in SDF."
```

Comment:
This seems like an error in our SDF file. Can we remove the `alwaysOn`?

Reply:
Removed in https://github.com/gazebosim/sdformat/pull/1294/commits/03b9864737974dcf5b62751d6b66b8dfaf571c3f.